packages/mobile/e2e/src/usecases/NewAccountPhoneVerification.js

```diff
@@ -0,0 +1,252 @@
+          await element(by.id(`VerificationCode${i + 1}`)).replaceText(secondCodeSet[i])
+        }
+
+        // Assert we've arrived at the home screen
+        await waitFor(element(by.id('SendOrRequestBar')))
+          .toBeVisible()
+          .withTimeout(30 * 1000)
+
+        // Assert that phone verification CTA is NOT served
+        try {
+          await element(by.id('CTA/ScrollContainer')).scroll(500, 'right')
+        } catch {}
+        await expect(element(by.text('Confirm Now'))).not.toExist()
+
+        // Assert that correct phone number is present in sidebar
+        await element(by.id('Hamburger')).tap()
+        await expect(element(by.text(`${examplePhoneNumber}`))).toBeVisible()
+      })
+    } else {
+      it('Then should handle when Komenci is down', async () => {
+        // Continue To Komenci down view
+        await element(by.text('Continue')).tap()
+
+        // Continue to Home Screen
+        await element(by.text('Continue')).tap()
+
+        // Assert we've arrived at the home screen
+        await waitFor(element(by.id('SendOrRequestBar')))
+          .toBeVisible()
+          .withTimeout(10 * 1000)
+
+        // Assert that phone verification CTA is served
+        try {
+          await element(by.id('CTA/ScrollContainer')).scroll(500, 'right')
+        } catch {}
+        await expect(element(by.text('Confirm Now'))).toExist()
+
+        // Assert that correct phone number is present in sidebar
+        await element(by.id('Hamburger')).tap()
+        await expect(element(by.text(`${examplePhoneNumber}`))).toBeVisible()
+      })
+    }
+  }
+
+  // This test is needed as a fall back if no previous tests run - empty specs are not allowed
+  // Assert correct content is visible on the phone verification screen
+  it('Then should have correct phone verification screen', async () => {
+    await dismissBanners()
+    await expect(element(by.text('Connect your phone number'))).toBeVisible()
+    let skipAttributes = await element(by.text('Skip')).getAttributes()
+    jestExpect(skipAttributes.enabled).toBe(true)
+    await waitFor(element(by.text('Do I need to confirm?')))
+      .toBeVisible()
+      .withTimeout(10000)
+
+    // Tap 'Do I need to confirm?' button
+    await element(by.text('Do I need to confirm?')).tap()
+
+    // Assert modal content is visible
+    await expect(element(by.text('Phone Numbers and Valora'))).toBeVisible()
+    await expect(
+      element(
+        by.text(
+          'Confirming makes it easy to connect with your friends by allowing you to send and receive funds to your phone number.\n\nCan I do this later?\n\nYes, but unconfirmed accounts can only send payments with QR codes or Account Addresses.\n\nSecure and Private\n\nValora uses state of the art cryptography to keep your phone number private.'
+        )
+      )
+    ).toBeVisible()
+
+    // Assert able to dismiss modal and skip
+    let modalDismissAttributes = await element(by.text('Dismiss')).getAttributes()
```

Comment:
What's the point of checking if the buttons are enabled? If we try to tap them and it doesn't work then the test will fail anyways, right?

Reply:
Removed asserts for buttons we tap, but left them for buttons we don't tap.